vtr_flow/scripts/vtr-flow.py

```diff
@@ -173,7 +173,30 @@ def vtr_command_argparser(prog=None):
                                type=float,
                                metavar="TIMEOUT_SECONDS",
                                help="Maximum time in seconds to spend on a single stage.")
-
+    #
+    # Iterative black-boxing flow arguments
+    #
+    iterative = parser.add_argument_group("Iterative", description="Iterative black-boxing flow for multi-clock circuits options")
+    iterative.add_argument("-iterative_bb",
+                            default=False,
+                            action="store_true",
+                            dest="iterative_bb",
+                            help="Use iterative black-boxing flow for multi-clock circuits")
+    iterative.add_argument("-once_bb",
```

Comment:
If we are to port this over, I believe the three method were there just for experimenting, we settled on the itterative_bb method. 

side note: the blanket bb was black boxing all the latches (blanket black boxing).

Reply:
If I understand correctly, I should remove all  the options and just use the itterative_bb setting? Iterative_bb is the default currently.

Reply:
See my comment below.  I think we should still be able to turn this on and off for experiments (justification in comment below) so I'd leave the capability in, but it should be iterative_bb by default.

Reply:
This has been fixed.